snippets/xen_dom0/snippet.yml

```diff
@@ -7,9 +7,9 @@ boards:
   qemu_cortex_a53:
     append:
       EXTRA_DTC_OVERLAY_FILE: boards/qemu_cortex_a53.overlay
-  rcar_h3ulcb_ca57:
+  rcar_h3ulcb/r8a77951/a57:
     append:
-      EXTRA_DTC_OVERLAY_FILE: boards/rcar_h3ulcb_ca57.overlay
+      EXTRA_DTC_OVERLAY_FILE: boards/rcar_h3ulcb_r8a77951_a57.overlay
   rcar_salvator_xs_m3:
```

Comment:
This part should be also changed to rcar_salvator_xs/r8a77961 (same for tests yaml's)

Reply:
Updated